pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +928,16 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	ContainerAlignedComputeResources = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           ContainerAlignedComputeResourcesNameKey,
+			Help:           "Cumulative number of aligned compute resources allocated to containers by alignment type.",
```

Comment:
are we accounting for a failed admissions somehow? Or only for successful?

Reply:
yes, using `CPUManagerPinningErrorsTotal` and `TopologyManagerAdmissionErrorsTotal`